arm-eventhub/2017-04-01/swagger/EventHub.json

```diff
@@ -0,0 +1,1930 @@
+        ],
+        "operationId": "EventHubs_Delete",
+        "x-ms-examples": {
+          "EventHubDelete": { "$ref": "../examples/EventHubs/EHEventHubDelete.json" }
+        },
+        "description": "Deletes an Event Hub from the specified Namespace and resource group.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639496.aspx"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/eventHubNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No content."
+          },
+          "200": {
+            "description": "Event Hub successfully deleted."
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "EventHubs"
+        ],
+        "operationId": "EventHubs_Get",
+        "x-ms-examples": {
+          "EventHubGet": { "$ref": "../examples/EventHubs/EHEventHubGet.json" }
+        },
+        "description": "Gets an Event Hubs description for the specified Event Hub.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639501.aspx"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/eventHubNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the Event Hub description.",
+            "schema": {
+              "$ref": "#/definitions/Eventhub"
+            }
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/namespaces/{namespaceName}/eventhubs/{eventHubName}/authorizationRules": {
+      "get": {
+        "tags": [
+          "EventHubs"
+        ],
+        "operationId": "EventHubs_ListAuthorizationRules",
```

Comment:
See your usage here: https://github.com/Azure/azure-rest-api-specs/pull/1305/files#diff-b4834156976cce96ec61cb634783f253R1315 .

This is good naming, it follows the `ListBy<PARENTRESOURCE>` convention. Please follow the same format in other places where you have `List` operation for consistency.